### DISCUSSION TITLE
1.3.3.8 testing

### DISCUSSION THREAD

**Discussion by TheWebMachine** [Guilouz/Creality-Helper-Script-Wiki#470] [General]
Has anyone confirmed there are no breaking changes in 1.3.3.8? I'd hate to have to roll back to 1.3.3.5 and set everything up a 3rd (10th) time if no one has actually tested 1.3.3.8 out fully. I just got everything how I want it...again. 🤣

Edit: My current list of identified issues as of 2024-03-23:

- Wifi has been funky since day one, both FW versions (1.3.3.5 also affected). I live in a dense suburban area and 2.4ghz spectrum is filled to the brim. It seems if you use a channel other than 1, 6, or 11, the wifi becomes unstable at any distance from the AP; a mere 3 meters/10ft in my case! Using 1, 6 or 11 is difficult for me here because those are defaults for most routers and most folks never change it. I had been using channels 3 and 9 for a long time in my house and only this new K1C had a problem with it. Upon switching the closer AP to Ch1, despite being far more crowded by my neighbors, the wifi has been stable, albeit slower due to the congestion. I'm so sick of new devices only being 2.4Ghz. 🤬

- Screen freeze during print only since 1.3.3.8, doesn't happen on 1.3.3.5 or if I use Guppy Screen, which I'm now doing.

- M600 or pause/resume since 1.3.3.8 doesn't keep hotend at temp when resuming print. It will heat for the filament swap, then cool down before attempting final purge and resume of print. As noted above, I'm using Guppy to get around the screen freeze issue.

It would be a huge help if Creality's changelog was a little more verbose than "some bug fixes." I hate that lazy BS. How the hell is anyone supposed to troubleshoot for breaking changes when they won't tell us what they broke!? 🤦🏻‍♂️

**Comment by halTWO3** [comment]
The firmware came out around 12 midnight last night, can't call it tested within that time range. However, I did install it and did a 6.5hr print without issues. If it's not broken, don't fix it if ur printer is running stable.

**Comment by TheWebMachine** [comment]
Oh, fair enough. I am using Guppy and only heard about 1.3.3.8 via github notifications. Wasn't sure when it came out. My apologies. 

I'm hoping Creality addressed a few random bugs in 1.3.3.5, so I'm down to give it a try if it has seen some successful usage thus far. 6.5hr print seems pretty stable to me. Do you use things like moonraker timelapse and the enhanced input shaping?

**Comment by halTWO3** [comment]
Oh np! And yes!  Rooted with moonraker, fluidd, kamp, entware, timelapse and guppyscreen.  Then removed creality web services.

Mine was more of an accidental update but turned out fine.  Was rolling fw v1.3.2.0 without issues.

**Comment by sweelssen** [comment]
I have problems with the install of moonraker, entware and mainsail or fluidd. i just got in my K1 Max and installed moonraker, entware and mainsail, but when i want the to install camera and settings (17 and 18 in helper script) moonraker  says: new hardware camera, not compatible, so i loose the monitoring. back to non rooted original 1.3.3.8. firmware

**Reply by joseb552** [reply]
When I rooted my K1 Max a couple of weeks ago I also had that camera hardware message but that was just for the advanced settings. I was able to enable the camera within Fluidd and have been using it since. I'll try to remember where I enabled it but it was a simple setting somewhere that enabled it. 

** Edit ** 

https://guilouz.github.io/Creality-K1-Series/configurations/configure-camera/

**Comment by Snjperman** [comment]
Had same issue, but continued anyway, went and added the camera manually which worked...later on after running shaper and pid ect and started printing I saw 2 cameras...had to delete the one I added 🤷‍♂️

**Comment by weston-wright** [comment]
I reset my K1 Max, upgraded to the new firmware, and have now tried to run the helper script. It keeps either timing out or just generally getting hung up on something. I hadn't had this happen before so I might roll back to 1.3.3.5

**Reply by weston-wright** [reply]
Quick update on this, I was able to eventually install all the items from the helper script. Not sure if it was on my end or what but finally got everything to install.

**Comment by TheWebMachine** [comment]
Got my K1C all upgraded and everything reinstalled.

Creality did not fix the finicky wifi in 1.3.3.8, like I had hoped they would. Thankfully, I'm a network engineer and was able to finesse my APs to cater to Creality's shoddy wifi implementation. When the hell are companies going to stop putting 2.4G wifi in their products!? Everyone and their mother is clogging up that entire band and, yet, they keep providing it as the only connectivity option. It's getting old...literally. (I partly blame Expressif for their hand in holding the industry back, but I digress.)

They also introduced a new bug...the screen will sometimes refuse to turn back on after it turns off during a print. It takes a full reboot to get it back on. I've disabled the screen timeout until I can get a handle on if it is, for sure, Creality's update to blame. I'm using stock screen software, so it's probably on their end of things because it was fine in 1.3.3.5.

Beyond that, everything else seems to be working the same as it did on 1.3.3.5. I'm sure Creality squashed a few bugs *somewhere*, but they always seems to suffer from a "two steps forward, two steps back" condition with many of their updates. 🤷🏻‍♂️

**Comment by Snjperman** [comment]
forgot to mention i did have issues with obico, couldnt get it to link for whatever reason.

**Comment by TheWebMachine** [comment]
Okay, so the screen isn't just freezing when off...it's just freezing during print. Is anyone else getting this? Set screen off timeout to disabled and run a job. At some random point, usually near the beginning of the print, the display freezes, stops updating and becomes unresponsive to touch.

I'll roll back to 1.3.3.5 later tonight and confirm the issue doesn't exist there, but I don't recall ever having the screen freeze on that version. I want to finish my current project first...about 7hrs/2 more print jobs to go. haha

**Reply by halTWO3** [reply]
What slicer do you use? Just wondering...

**Reply by TheWebMachine** [reply]
I also use Prusaslicer. That's why I asked if you were sure it was the cause in the other discussion. haha Prusaslicer doesn't cause me the issue on 1.3.3.5. I am using 96x96/PNG, 300x300/PNG for thumbnails, as that is what I saw in my teardown of a Creality Print sliced gcode file.

**Reply by halTWO3** [reply]
Forgot to add that it only happens if I print something large like say 5"x6" object.  Do you use kamp?

**Reply by TheWebMachine** [reply]
Probably not, since I don't think I know what kamp is. haha

**Reply by halTWO3** [reply]
Ah np. I also use 96x96/300x300.  If you print and the screen freezes, try setting thumbnails to 48/48. It works until a certain height then starts freezing again. 

Should check kamp out. I think it actually works. I use it on all my klipper printers. Helps me get perfect first layers. I moved my settings to orca but been using ps for a long time, can't... let... go.

**Reply by pad7r1x** [reply]
i use 45x45, 300x300 for thumbnails and it works fine.

**Reply by halTWO3** [reply]
Tested mentioned sized, still same issue.

**Comment by TheWebMachine** [comment]
> It works until a certain height then starts freezing again.
 I wonder if it has to do with that fill color transparency thing the Creality interface tries to do to show progress.

Oh! You mean the Klipper Adaptive Mesh and Purge. Gotcha! No, I don't use it.

**Comment by Horgul123** [comment]
Hi, 
any more tests with the latest FW?

I need to reset my K1 to install the latest Guilouz Script and want to make sure, that installing the latest FW is safe enough.

**Reply by halTWO3** [reply]
Just finished my third 16hr print on the latest fw.     So for me, I think it's safe. Root script works like a charm.

**Reply by TheWebMachine** [reply]
Overall, it is safe with one notable exception:

A bug with resuming paused prints/M600 is still being tested out.

Beyond that, a minor bug:

If you use Prusaslicer with thumbnails and stock Creality screen software, the screen software locks up during the print until hard reset (does not interfere with print or web interfaces).

Otherwise, 1.3.3.8 is as stable for printing as 1.3.3.5. I honestly can't figure out exactly what all they changed between the two versions. It doesn't appear to be anything major, but it also doesn't seem like they actually fixed anything, either. Haha

**Comment by Schulli73** [comment]
So I'm going back to 1.3.3.5

**Reply by TheWebMachine** [reply]
What was the last straw for you?

**Comment by sweelssen** [comment]
weird, my problem with moonraker and mainsail, that left me without the camera solved itself, now i can add the K1 max camera. the printer has been on for 3 days or so, i printed with several brands of pla filaments. some problems with the tree support fusing with the object, solved that by setting the support/object xy distance to 0.7. The 1.3.3.8 rooted firmware seems to be working fine.

**Comment by dingausmwald** [comment]
So, any news on the screen thing or anything? It seems like this is kinda dead. Over 6 months and there are still major bugs on the machine everybody is living with?

For me, and in every k1 (not MAX) config_example the factory_printer.cfg is for the max version. Everybody blindly resetting their settings get woken up kindly by bumping heads as the size is set to 305 and god knows what else

the screen freezes on every other print making it a ritual to restart the printer after every job.

On filament change (quit material, load material) the temperature stays high forever.

M600 is buggy.. and so on.
Don't want to rant. Maybe there is another place solutions are discussed? Am i missing something?

**Reply by gunrunnerjohn** [reply]
Have you tried Reddit?

https://www.reddit.com/r/Creality/
